client/js/components/admin/instances/instance_list.jsx

```diff
@@ -7,7 +7,7 @@ export default class InstanceList extends React.Component {
     instances: React.PropTypes.shape({}).isRequired,
   };
 
-  deleteInstance() {
+  static deleteInstance() {
```

Comment:
refactor to be static as it doesn't use `this`. Complies with http://eslint.org/docs/rules/class-methods-use-this